src/ResourceManager/Network/Commands.Network/help/Get-AzureRmBgpServiceCommunity.md

```diff
@@ -16,15 +18,15 @@ Get-AzureRmBgpServiceCommunity [-DefaultProfile <IAzureContextContainer>] [<Comm
 ```
 
 ## DESCRIPTION
+This cmdlet provides a list of all services / regions, BGP communities, and associated prefixes.
 
 ## EXAMPLES
 
 ### Example 1
 ```
-PS C:\> {{ Add example code here }}
+Get-AzureRmBgpServiceCommunity
```

Comment:
@ganesr would you be able to add the corresponding output (or a snippet of it if it's too big) from this cmdlet execution in this code block?